AssetInformationApi.Tests/V1/Boundary/Request/Validation/AddAssetRequestValidatorTests.cs

```diff
@@ -9,6 +9,7 @@
 
 namespace AssetInformationApi.Tests.V1.Boundary.Request.Validation
 {
+#nullable enable
```

Comment:
What is this for?

Reply:
This is so we can use the nullable string annotation here:
`public void RequestShouldNotErrorWhenPostcodeIsEmptyOrNullForTemporarysAccommodationAsset(string? postcode)`

It serves no other purpose really and doesn't affect the functionality itself.